chain/messagepool/block_proba_test.go

```diff
@@ -0,0 +1,8 @@
+package messagepool
+
+import "testing"
+
+func TestBlockProbability(t *testing.T) {
+	mp := &MessagePool{}
+	t.Logf("%+v\n", mp.blockProbabilities(1-0.15))
```

Comment:
not much of a test

Reply:
I think kuba wanted to eyeball it, but we can probably turn it into a real test :)

Reply:
I turned it into an actual test.